optimum/onnxruntime/modeling_ort.py

```diff
@@ -267,10 +267,14 @@ def __init__(
             **kwargs,
         )
 
-        self.inputs_names = {input_key.name: idx for idx, input_key in enumerate(model.get_inputs())}
+        # why are these not lists ?
+        self.input_names = {input_key.name: idx for idx, input_key in enumerate(model.get_inputs())}
```

Comment:
Changing the variable names makes the diff for this PR bigger, which makes it a bit harder to review

Reply:
yeah the issue is that there were naming discrepancies which made little sense to me, especially between `ORTModel` and `ORTModelPart`
https://github.com/huggingface/optimum/blob/ac951ca0ae49983b1515b84342fea15c3b7ec35c/optimum/onnxruntime/modeling_ort.py#L270
https://github.com/huggingface/optimum/blob/ac951ca0ae49983b1515b84342fea15c3b7ec35c/optimum/onnxruntime/base.py#L54
is that intended ?
I think there might be a way to remove these naming changes, which is by fixing dtype in the seq2seq modeling classes instead of model parts. tell me if that makes more sense to you.

Reply:
most likely not